bonus.md

```diff
@@ -4,10 +4,28 @@ Edit this file to answer the following questions. Give examples. You may need to
 
 1. Why is Hash.fetch useful?
 
+Hash.fetch is useful because it allows us to set up a default value if nothing is passed in on initialize.
```

Comment:
What's the difference between `args[key]` and `args.fetch(key)` if the key `key` is not present in the `args` hash?